File: reference/stack/2390-Removing-Stars-From-a-String.py
class Solution:
    def removeStars(self, s: str) -> str:
        stacks = []
        for c in s:
            if stacks and c == "*" and stacks[-1] != "*":
                stacks.pop()
            else:    
                stacks.append(c)
        return "".join(stacks)

File: reference/stack/2390-Removing-Stars-From-a-String.go
func removeStars(s string) string {
    var stacks []rune
    for _, c := range s {
        if len(stacks) > 0 && c == '*' && stacks[len(stacks) - 1] != '*' {
            stacks = stacks[:len(stacks) - 1]
        } else {
            stacks = append(stacks, c)
        }
    }

    return string(stacks)    
}